server/utils/formatter.js

```diff
@@ -104,9 +107,11 @@ export const paginationResponse = (res,
   numberOfPages,
   data,
   nextPage,
-  prevPage) => res.status(200).json({
+  prevPage) => {
```

Comment:
Might be cool to take off the brackets and return statement here since its an arrow function that immediately returns a value